### PR TITLE
Backend #42 task

### LINKED ISSUE #42: (SP: 3) Implement stepper Countries location service
## Description

For [issue](https://github.com/WebUI-Project-Based-07-vk/Client/issues/67) implement location service, to fetch countries by using 
[countrystatecity.in](https://countrystatecity.in/) API

## Implement

- [ ] Service
- [ ] Controller
- [ ] Route
- [ ] Unit tests
- [ ] Swagger docs

### PR DESCRIPTION
**Pull request by SeredenkoDS**
With the new changes, localhost:8080/get-country-list uses countrystatecity API to retrieve the list of countries. Unit test is not added (I guess I will make a separate task for that)

### CODE REVIEW

File: src/controllers/locationServiceCountries.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+require('dotenv').config({ path: '~/../.env.local' })
+const locationServiceCountries = require('~/services/locationServiceCountries')
+
+const getCountryList = async (req, res) => {
+  const country_list = await locationServiceCountries.getCountries(process.env.COUNTRYSTATECITY_API_KEY)
```
Comment (tufned) [inline src/controllers/locationServiceCountries.js]: Please, can you move `process.env.COUNTRYSTATECITY_API_KEY` to the config.js ?

File: src/services/locationServiceCountries.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+const locationServiceCountries = {
+  getCountries: async (api_key) => {
+    //console.log(api_key);
```
Comment (tufned) [inline src/services/locationServiceCountries.js]: Remove `console.log()`

**Review by tufned — CHANGES_REQUESTED** [review]
Overall looks good!
Just added a few notes to improve the code

**Comment by SeredenkoDS** [comment]
Hey Den, thanks for the comment! Made the requested changes.

**Comment by SeredenkoDS** [comment]
> Overall looks good! Just added a few notes to improve the code

Hey Den, thanks for the comment! Made the requested changes.

**Review by tufned — APPROVED** [review]

**Review by olehpona — APPROVED** [review]

**Review by MrAK-dev — APPROVED** [review]

File: src/services/locationServiceCountries.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      redirect: 'follow'
+    }
+
+    try {
+      const response = await fetch('https://api.countrystatecity.in/v1/countries', requestOptions)
+      const result = await response.text()
+      return JSON.parse(result)
+    } catch (error) {
+      //
+    }
```
Comment (vlad-khrychov) [inline src/services/locationServiceCountries.js]: Why we need empty `catch`? Maybe in this case we should throw another Error with intimation to the user?

**Review by vlad-khrychov — CHANGES_REQUESTED** [review]
Are you planning to add tests? Or create a new issue and ask teammates to help with testing if they have time.

File: src/services/locationServiceCountries.js (hunk 3 above)
Comment (SeredenkoDS) [reply src/services/locationServiceCountries.js]: Hey Vlad, made the requested changes to the 'catch' block. Will do the same for #43 shortly. As for the tests, a separate issue #62 has been created.

**Review by vlad-khrychov — APPROVED** [review]

**Pull request merged**